compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/ExposedDropdownMenu.skiko.kt

```diff
@@ -0,0 +1,713 @@
+            disabledBorderColor = disabledBorderColor,
+            errorBorderColor = errorBorderColor,
+            focusedLeadingIconColor = focusedLeadingIconColor,
+            unfocusedLeadingIconColor = unfocusedLeadingIconColor,
+            disabledLeadingIconColor = disabledLeadingIconColor,
+            errorLeadingIconColor = errorLeadingIconColor,
+            focusedTrailingIconColor = focusedTrailingIconColor,
+            unfocusedTrailingIconColor = unfocusedTrailingIconColor,
+            disabledTrailingIconColor = disabledTrailingIconColor,
+            errorTrailingIconColor = errorTrailingIconColor,
+            focusedLabelColor = focusedLabelColor,
+            unfocusedLabelColor = unfocusedLabelColor,
+            disabledLabelColor = disabledLabelColor,
+            errorLabelColor = errorLabelColor,
+            focusedPlaceholderColor = focusedPlaceholderColor,
+            unfocusedPlaceholderColor = unfocusedPlaceholderColor,
+            disabledPlaceholderColor = disabledPlaceholderColor,
+            errorPlaceholderColor = errorPlaceholderColor,
+            focusedPrefixColor = focusedPrefixColor,
+            unfocusedPrefixColor = unfocusedPrefixColor,
+            disabledPrefixColor = disabledPrefixColor,
+            errorPrefixColor = errorPrefixColor,
+            focusedSuffixColor = focusedSuffixColor,
+            unfocusedSuffixColor = unfocusedSuffixColor,
+            disabledSuffixColor = disabledSuffixColor,
+            errorSuffixColor = errorSuffixColor,
+        )
+}
+
+@Suppress("ComposableModifierFactory")
+@Composable
+private fun Modifier.expandable(
+    expanded: Boolean,
+    onExpandedChange: () -> Unit,
+    menuDescription: String = getString(Strings.ExposedDropdownMenu),
+    expandedDescription: String = getString(Strings.MenuExpanded),
+    collapsedDescription: String = getString(Strings.MenuCollapsed),
+) = pointerInput(Unit) {
+    awaitEachGesture {
+        // Must be PointerEventPass.Initial to observe events before the text field consumes them
+        // in the Main pass
+        awaitFirstDown(pass = PointerEventPass.Initial)
+        val upEvent = waitForUpOrCancellation(pass = PointerEventPass.Initial)
+        if (upEvent != null) {
+            onExpandedChange()
+        }
+    }
+}.semantics {
+    stateDescription = if (expanded) expandedDescription else collapsedDescription
+    contentDescription = menuDescription
+    onClick {
+        onExpandedChange()
+        true
+    }
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+private fun updateHeight(
+    windowInfo: WindowInfo,
+    coordinates: LayoutCoordinates?,
+    verticalMarginInPx: Int,
+    onHeightUpdate: (Int) -> Unit
+) {
+    coordinates ?: return
+    val boundsInWindow = coordinates.boundsInWindow()
+    val visibleWindowBounds = windowInfo.size.toIntRect()
+    val heightAbove = boundsInWindow.top - visibleWindowBounds.top
+    val heightBelow = visibleWindowBounds.height - boundsInWindow.bottom
```

Comment:
```suggestion
    val heightBelow = visibleWindowBounds.bottom - boundsInWindow.bottom
```
It seems is more correct here, as we work in bounds instead of sizes

Reply:
No, it should be `bottom - top` ( == `height`)
https://github.com/JetBrains/compose-multiplatform-core/blob/8ee0520524968db47aeb702dae7895db18268225/compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/ExposedDropdownMenu.kt#L1053